core-services/prow/02_config/_plugins.yaml

```diff
@@ -2851,6 +2851,12 @@ config_updater:
         app.ci:
         - ci
       name: image-mirror-mappings
+    core-services/prow/02_config/**/_prowconfig.yaml:
+      clusters:
+        app.ci:
+        - ci
+      name: config
```

Comment:
```
$ oc get cm -n ci config
NAME     DATA   AGE
config   1      368d
```

Name conflicting?

Reply:
That is intentional, it all needs to go into the same configmap